JCA Latest Update

Azodion opened the pull request:
## What type of PR is this?
1. [ ] Bug
2. [x] Change
3. [ ] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:
Added new items from JCA Infantry Arsenal Updates.


### Please specify which Issue this PR Resolves (If Applicable).

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:

********************************************************
Notes:

Azodion committed:
JCA Latest Update

wersal454 commented:
Grave is probably gonna add this https://steamcommunity.com/sharedfiles/filedetails/?id=3477061208 to the arsenal, so store will need to be updated again

Azodion commented:
no biggie. will add it when it gets updated again

Azodion committed:
Mk23 Update

Forgot to add the M9A1 before. So added that along with some magazines I forgot about.

wersal454 commented:
I don't remember if we have support for https://steamcommunity.com/sharedfiles/filedetails/?id=3473383676 (maybe we do) could you also add this?

Azodion commented:
Was thinking about adding it. I definitely can.

UAC-MaxxLite reviewed (dismissed):
lgtm

Azodion commented:
gonna be adding the Infantry Equipment to this PR in a few minutes :)

Azodion committed:
JCA Infantry Equipment

UAC-MaxxLite reviewed (approved):
Lgtm.
